package.json

```diff
@@ -30,7 +30,7 @@
     "canonical-path": "0.0.2",
     "del": "^1.2.0",
     "dgeni": "^0.4.0",
-    "dgeni-packages": "^0.11.0",
+    "dgeni-packages": "^0.11.1",
```

Comment:
there was a bug in the new inline tag def processor that we need for the target package and is fixed in 0.11.1